src/components/atoms/forms/text-field.scss

```diff
@@ -0,0 +1,31 @@
+@use "../../../sass/mixins";
+@use "../../../sass/settings";
+@use "../../../sass/typography";
+
+.text-field,
+.text-field[type="text"],
+.text-field[type="email"] {
+  appearance: none;
+}
+
+.text-field {
+  @include typography.font-size-and-vertical-height(settings.$font-size-base-in-px);
+  @include mixins.margin(settings.$site-spacing / 4, "bottom");
+  @include mixins.padding(settings.$site-spacing - 1 12);
+
+  border: 1px solid settings.$divider;
+  border-radius: 3px;
+  color: settings.$color-text;
+  display: block;
+  font-family: settings.$font-family-secondary;
+  width: 100%;
+
+  &:focus {
+    border: 1px solid settings.$color-text-placeholder;
+    outline: 0;
```

Comment:
removing this will remove the browsers default focus style. Also in Safari it will not follow the shape of the border-radius